src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/ThemeChange.cs

```diff
@@ -0,0 +1,31 @@
+#if ANDROID || IOS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class ThemeChange : _IssuesUITest
+	{
+		public override string Issue => "UI theme change during the runtime";
+
+		public ThemeChange(TestDevice device) : base(device)
+		{
+		}
+
+		[Test]
+		public void AppThemeShouldChange()
+		{
+			_ = App.WaitForElement("labelVisibleOnlyInLightMode");
+
+			App.SetDarkTheme();
+			_ = App.WaitForElement("labelVisibleOnlyInDarkMode");
+			VerifyScreenshot();
```

Comment:
Just updated the test, taking just one snapshot, will use the test name to save the png file. But, when we require several snapshots in the tests, must use different names.

Reply:
Thank you!